spec/rubocop/plugin_spec.rb

```diff
@@ -51,4 +40,33 @@
       end
     end
   end
+
+  context 'integration', :isolated_environment do
+    include_context 'cli spec behavior'
+
+    let(:rubocop) { "#{RuboCop::ConfigLoader::RUBOCOP_HOME}/exe/rubocop" }
+
+    context 'without bundler', :isolated_bundler do
```

Comment:
Is `isolated_bundler` actually necessary here? The tests pass (including `bundle exec rake default`) without it, and with it, the gems are missing so rubocop fails to load.

It's possible there's some rspec seed that fails without it though, I don't know.

Reply:
Actually hmm that will prevent the test from actually testing when the gem is not loaded since it is in the Gemfile...

Could you create a plugin in a separate file and have it require that?

Reply:
Without bundler is needed, since otherwise it will be loaded. That was a bit surprising to me since the new process is started with just `ruby`, but there is an environment variable being passed along that causes ruby to load bundler regardless, since the current process is already started with bundler.

I'm unsure why the test fails in CI. The gems are installed, so should be available to require? Same as locally.